### PR TITLE
Add ability to have a simple install

### LINKED ISSUE #98: Simple vs custom template option
We should consider a simple vs custom template option. This would allow new users to just get started with minimal effort and allow others to go through the full question set to customize their project. 

This tooling in Copier may help with that: https://copier.readthedocs.io/en/stable/configuring/#include-other-yaml-files

### PR DESCRIPTION
**Pull request by jeremykubica**
Closes #98 

Creates a "simple" install option that uses the default values for linting, isort, and create example module.

### CODE REVIEW

**Review by delucchi-cmu — APPROVED** [review]
This seems like a great idea!

**Comment by jeremykubica** [comment]
All credit goes to Drew for the idea.

File: copier.yml [hunk 1]
```diff
@@ -43,6 +51,7 @@ preferred_linter:
         pylint: pylint
         black: black
         none: none
+    when: "{{ custom_install }}"
```
Comment (drewoldag) [inline copier.yml:54]: This is so straightforward - perfect!

**Review by drewoldag — COMMENTED** [review]
This looks great, thank you!

**Pull request merged**